src/main/java/edu/agh/wfiis/designpatterns/patternsUsageExample/DesignPatternsManager.java

```diff
@@ -59,21 +43,24 @@ public String generateDesignPatternsCourseworkForStudents(){
         return "very difficult coursework";
     }
 
-}
-
-enum PatternName {
-    BUILDER, DECORATOR, STRATEGY;
-}
+    static class PatternHandler {
```

Comment:
Name of this class is not meaningful.  Single responsibility principle is broken.